apps/testnorge-skd/skd-core/src/main/java/no/nav/registre/skd/consumer/dto/Relasjon.java

```diff
@@ -0,0 +1,22 @@
+package no.nav.registre.skd.consumer.dto;
+
+import lombok.Builder;
+import lombok.Data;
+
+@Data
+@Builder
+public class Relasjon {
+
+    private final String kortnavn;
+    private final String datoDo;
+    private final String typeRelBeskr;
+    private final String mellomnavn;
+    private final String etternavn;
+    private final int adresseStatus;
```

Comment:
Denne er altid statt? Eller skal den kunne være null? Isåfall `private final Integer adresseStatus;`

Reply:
Dette er slik den er i HodejegerenConsumer klassen fra objektet. Men endrer til `Integer`